### PR TITLE
update MOC codes

### LINKED ISSUE #188: MOC missing or inaccurate
This is a collection of prior issues that are all related and I thought should be gathered in one place to fix together.  

Issue #80 - MOC 1110 Navy is currently listed as "Tomahawk Weapon Maintenance Technician" but a tester who was in Navy and was 1110 MOS is telling me that code is Surface Warfare Officer?

Issue #67 - valid MOC codes are not being found when i search Army 15W or Coast Guard 56; are all the MOC codes included in the database?

Issue #78 when searching for older MOC code (e.g., 31K - communications / wire specialist) no results are found. if this is the case for older MOC codes then the tool won't be as useful for older Veterans.

### PR DESCRIPTION
**Pull request by cfeeney-va**
Fixes #188

### CODE REVIEW

File: db/seed/data_importer.rb [hunk 1]
```diff
@@ -1,21 +1,34 @@
+require 'iconv' unless String.method_defined?(:encode)
+
 module DataImporter
 
   def self.import_all_mos
     csv_text = File.read('db/seed/Mil2FedJobsSupportFiles/mos_list.csv')
     csv = CSV.parse(csv_text, headers: true)
```
Comment (ayaleloehr) [inline db/seed/data_importer.rb]: Shouldn't these two lines be deleted?

**Comment by ayaleloehr** [comment]
This is looking excellent @cfeeney-va.  Let's have a quick chat tomorrow morning to help me better understand why there is both a seed and a migration for this and when each would be used. 

p.s. encoding issues stink.

**Comment by ayaleloehr** [comment]
Let's also delete the Mil2FedJobsSupportFiles folder. Those will still be in the git history if we ever need them.

**Comment by cfeeney-va** [comment]
you fixed it :)

**Comment by cfeeney-va** [comment]
per discussion with marina, updating the migration to a one-time rake task

**Comment by ayaleloehr** [comment]
LGTM.  Test failed because code coverage fell below 74.5%.  Suggest we just change that threshold to 70%. We made the group decision that we are at a decent code coverage and I'm not sure what adding a test for this one time rake task will buy us at this point.  

@cfeeney-va once you make that change and tests are green, feel free to merge!

**Comment by cfeeney-va** [comment]
thanks @ayaleloehr !

**Pull request merged**